ngraph/frontend/paddlepaddle/src/node_context.hpp

```diff
@@ -70,6 +73,10 @@ class NodeContext
             return false;
         }
     }
+
+    std::vector<OutPortName> get_output_names() const { return node.get_output_names(); }
+    NamedOutputs default_single_output_mapping(const std::shared_ptr<Node> &ngraph_node,
+                                               const std::vector<OutPortName>& pdpd_out_names) const;
```

Comment:
minor: should it be renamed to 'required_pdpd_out_names' for consistency?

Reply:
done